man/dataset_generate_reports.Rd

```diff
@@ -1,22 +0,0 @@
-% Generated by roxygen2: do not edit by hand
```

Comment:
Confirming that there is no longer an option to build reports for all studies automatically? Instead each dataset needs to be explicitly listed using the function `dataset_generate_report`. (I don't think this is a problem - and is probably very sensible as the number of datasets gets ever larger, but I'd previously occasionally rebuild all reports to have current copies.)

Reply:
Actually you can, just not a seperate function. Just add multiple values for `dataset_id`. For example

```
# 2 studies
dataset_gerenerate_report(c("Angevin_2011", "Bragg_2002", austraits)

# all studies
dataset_gerenerate_report(dirs("data"), austraits)
```

Reply:
Excellent. Good to be reminded of the ```dirs("data")``` code.